keda/templates/manager/role.yaml

```diff
@@ -17,11 +17,13 @@ rules:
   resources:
   - secrets
   verbs:
+  {{- if not .Values.certificates.certManager.enabled }}
```

Comment:
I guess that with my previous comment, the condition here should be if self generated AND NOT cert manager